dali/python/backend_impl.cc

```diff
@@ -1117,6 +1118,22 @@ PYBIND11_MODULE(backend_impl, m) {
 
   m.def("GetCxx11AbiFlag", &GetCxx11AbiFlag);
 
+  m.def("HasCudaContext", []{
+    return DeviceGuard{}.has_old_context();
```

Comment:
Don't abuse DeviceGuard for this.
```suggestion
    DALI_ENFORCE(cuInitChecked(),
      "Failed to load libcuda.so. "
      "Check your library paths and if the driver is installed correctly.");
    CUcontext ctx;
    CUDA_CALL(cuCtxGetCurrent(&ctx));
    return ctx != nullptr;
```

Reply:
done